README.md

```diff
@@ -52,25 +53,24 @@ beta = np.array(beta.todense())
 
 # training data
 Xr = np.random.normal(0.0, 1.0, [n_samples, n_features])
-yr = model.simulate(beta0, beta, Xr)
+yr = glm.simulate(beta0, beta, Xr)
 
 # testing data
 Xt = np.random.normal(0.0, 1.0, [n_samples, n_features])
-yt = model.simulate(beta0, beta, Xt)
+yt = glm.simulate(beta0, beta, Xt)
 
 # fit Generalized Linear Model
 scaler = StandardScaler().fit(Xr)
-model.fit(scaler.transform(Xr), yr)
+glm.fit(scaler.transform(Xr), yr)
 
 # we'll get .fit_params after .fit(), here we get one set of fit parameters
-fit_param = model[-1].fit_
+fit_param = glm[-1].fit_
 
 # we can use fitted parameters to predict
-yhat = model.predict(scaler.transform(Xt))
+yhat = glm.predict(scaler.transform(Xt))
 ```
 
-See full example on how to use `pyglmnet`
-[here](http://pavanramkumar.github.io/pyglmnet/auto_examples/plot_poisson.html)
+[See the full example on how to use `pyglmnet`](http://pavanramkumar.github.io/pyglmnet/auto_examples/plot_poisson.html)
```

Comment:
@jasmainak i think it's a grammar edit! i'll raise a separate issue for relative paths